src/Nethermind/Nethermind.Abi/AbiDefinition.cs

```diff
@@ -65,3 +74,70 @@ public void Add(AbiErrorDescription @error)
         public static string GetName(string name) => char.IsUpper(name[0]) ? char.ToLowerInvariant(name[0]) + name[1..] : name;
     }
 }
+
+
+namespace Nethermind.Blockchain.Contracts.Json
```

Comment:
I'd appreciate if you could keep the file the same. This PR is large enough as it is without extra refactor.